myst_nbs/generalized_linear_models/GLM-poisson-regression.myst.md

```diff
@@ -6,15 +6,40 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: pymc_env
+  display_name: pymc-ex
   language: python
-  name: pymc_env
+  name: pymc-ex
 ---
 
 +++ {"papermill": {"duration": 0.043172, "end_time": "2021-02-23T11:26:55.064791", "exception": false, "start_time": "2021-02-23T11:26:55.021619", "status": "completed"}, "tags": []}
 
+(GLM-poisson-regression)=
 # GLM: Poisson Regression
 
+:::{post} November 12, 2022
+:tags: regression, poisson
+:category: Intermediate
```

Comment:
The flying dove in the preview: https://pymcio--462.org.readthedocs.build/projects/examples/en/462/generalized_linear_models/GLM-poisson-regression.html is not green because tagss and categories are case sensitive. it should be everything lowercase.